giskard/rag/metrics/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .base import Metric
 from .correctness import CorrectnessMetric, correctness_metric
+# from .ragas_metrics import RagasMetric
```

Comment:
?

Reply:
Some redundant code found during the test and not deleted.